### PR TITLE
libxkbcommon: new package at 0.9.1

### PR DESCRIPTION
**Pull request by alaughlin**
## Description of package
https://github.com/xkbcommon/libxkbcommon

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libxkbcommon/build [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+mkdir build
```
Comment (dylanaraps) [inline community/libxkbcommon/build]: These two lines can be removed.

This is what the `atk` package looks like for reference:

```sh
export DESTDIR="$1"

meson \
    --prefix=/usr \
    --sysconfdir=/etc \
    --mandir=/usr/share/man \
    -Ddocs=false \
    -Dintrospection=false \
    . output

ninja -C output
ninja -C output install
```

File: community/libxkbcommon/depends [hunk 2]
```diff
@@ -0,0 +1,2 @@
+meson   make
+samurai make
```
Comment (dylanaraps) [inline community/libxkbcommon/depends]: This can be removed. `meson` pulls in `samurai` automatically.

Comment (alaughlin) [reply community/libxkbcommon/depends]: 👍

File: community/libxkbcommon/build (hunk 1 above)
Comment (alaughlin) [reply community/libxkbcommon/build]: fixed!

File: community/libxkbcommon/build [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+meson \
+    --prefix=/usr \
+    --sysconfdir=/usr \
```
Comment (dylanaraps) [inline community/libxkbcommon/build]: This and `--mandir` may not need adding if these values match the defaults.

Comment (alaughlin) [reply community/libxkbcommon/build]: you're right. fixed!

**Pull request merged**